vmdb/Gemfile

```diff
@@ -7,7 +7,7 @@ MiqBundler.include_gemfile("../lib/Gemfile", binding)
 # VMDB specific gems
 #
 
-gem "prototype-rails", "~>3.2.1"
+gem "jquery-rjs", "=0.1.1", :git => 'https://github.com/amatsuda/jquery-rjs.git'
```

Comment:
Just yesterday the build team identified solutions to handle much of the trouble introduced to the build process by git based gems. That being said it would still be valuable to avoid them where possible.